packages/electrode-archetype-react-app/lib/features.js

```diff
@@ -218,10 +218,18 @@ class Feature {
     }
     const dependencies = { ...pkg.dependencies };
     const devDependencies = { ...pkg.devDependencies };
-    if (enabled && this.electrodeOptArchetype.devOnly) {
+
+    let devOnly = true;
+    if (this.electrodeOptArchetype.devOnly !== undefined) {
+      devOnly = this.electrodeOptArchetype.devOnly;
+    } else if (this.package && this.package.electrodeOptArchetype.devOnly !== undefined) {
+      devOnly = this.package.electrodeOptArchetype.devOnly;
```

Comment:
what's `this.package` vs `this._package`?

Reply:
oh got it. nvm

Reply:
`_package` is the actual object containing the `package.json` contents (really, a cache if you will), and `package` is a getter that is capable of fetching the `package.json` contents and caching it to _package. No consumer of this class should access _package directly.